README/README-docker-image.md

```diff
@@ -1,20 +1,28 @@
-# Setting the project
+# IMPORTANT
+The correct way to use the docker container in this project is by using the docker-compose file.
```

Comment:
It's better to described as "the recommended way" or something similar. The use of the `Dockerfile` is not wrong and can be preferred by some developers that need deeper access to the container or simply like to work that way.